### PR TITLE
Vertically align remove button in facets to be in middle. Closes #210

### LINKED ISSUE #206: Use "Classic Catalog" instead of "Main Catalog" on record view page
screenshot from 2015-08-19 16 13 03

### LINKED ISSUE #208: Get "orangelight" out of title bar
title

Replace w/ what, I'm not sure, but I doubt LISSC etc. will go for OrangeLight.

### LINKED ISSUE #210: Line up facet value "x", and count, add label for "x"
facet

The "x" is high, and I also wonder if the count is higher. I can't tell.

Also, not to harp on tooltips, but I wonder if there are accessibility issues around the fact that what this does isn't labeled?

The HTML markup is this:

```html
<a class="remove" 
    href="/?q=john+cage+variations&amp;search_field=all_fields">
    <span class="glyphicon glyphicon-remove"></span>
    <span class="sr-only">[remove]</span>
</a>
```

I wonder if add `@data-original-title` and/or `@title` with the value 'Remove Format facet value "Book"' (or something like that) to the anchor tag that would be a good idea?

### LINKED ISSUE #213: Pivot Facets don't have 'x' button
The 'x' button has disappeared from the pivot facet panel.The only way to remove a pivot facet limit is in the render constraints bar at the top.
pivot_facet

### PR DESCRIPTION
**Pull request by axapu**
Get orangelight out of title bar and fix Guided search title. Closes #208

%~~Change main catalog to classic catalog. Closes #206~~

Closes #213.

### CODE REVIEW

**Comment by jpstroop** [comment]
I forget which it is now, but we should keep the label for WebVoyage that Marvin wanted, whatever it was. Is this PR doing that?

**Comment by axapu** [comment]
Is WebVoyage the name to replace Orangelight? I changed it to "Catalog | Princeton University Library"

**Comment by jpstroop** [comment]
No, sorry, WebVoyage is the current catalog.princeton.edu. Nikitas talked with Marvin directly and I think we need to stick with "Main Catalog" (as opposed to "Classic ...") for now. Is that correct, @tampakis?

**Comment by jpstroop** [comment]
ps: I'm cool with "Catalog | Princeton University Library"

File: config/locales/blacklight.en.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 en:
   blacklight:
-    application_name: 'Orangelight'
+    application_name: 'Catalog | Princeton University Library'
     welcome: 'Welcome!'
-    voyager: 'Main Catalog'
+    voyager: 'Classic Catalog'
```
Comment (tampakis) [inline config/locales/blacklight.en.yml]: This should stay Main Catalog

**Comment by axapu** [comment]
Okay. I'll revert that change. Close #206?

**Comment by jpstroop** [comment]
+1, just did

**Comment by tampakis** [comment]
:+1:

**Pull request merged**